docs/Gemfile.lock

```diff
@@ -0,0 +1,84 @@
+GEM
```

Comment:
Does the file `Gemfile.lock` need to be checked in?

When I `bundle install` in the dir, the following error message was got:

```
bundle install
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Bundler could not find compatible versions for gem "jekyll-feed":
  In snapshot (Gemfile.lock):
    jekyll-feed (= 0.16.0)

  In Gemfile:
    jekyll-feed (~> 0.12)

    github-pages was resolved to 36, which depends on
      jekyll-feed (= 0.2.3)

Deleting your Gemfile.lock file and running `bundle install` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
Bundler could not find compatible versions for gem "kramdown":
  In snapshot (Gemfile.lock):
    kramdown (= 2.4.0)

  In Gemfile:
    github-pages was resolved to 8, which depends on
      kramdown (= 1.0.2)

    jekyll-feed (~> 0.12) was resolved to 0.16.0, which depends on
      jekyll (>= 3.7, < 5.0) was resolved to 4.2.2, which depends on
        kramdown (~> 2.3)

Deleting your Gemfile.lock file and running `bundle install` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

I had to manually delete the file `Gemfile.lock` then run `bundle install` again.

Reply:
`Gemfile.lock` is removed from the repo.